e2e/workspace/crate2/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+use ts_rs::TS;
+
+#[derive(TS)]
+pub struct Crate2 {
+    pub x: i32
+}
```

Comment:
I believe I've got this to work when `#[export, export_to = "..."]` is used

Reply:
Yeah, if all crates of the workspace use `#[ts(export_to = "../some_dir")]`, then they all put their files in one directory & the imports work.

Reply:
Correct, and since they are all in the same workspace, it should be easy enough to ensure they are in the same directory.
Although, due to the changes to path diffing, I don't think they even **need** to be... just `#[ts(export)]` might be enough

Reply:
> just #[ts(export)] might be enough

Haven't tested this yet though

Reply:
> Haven't tested this yet though

Tested it now, it works!